eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -152,6 +160,7 @@ instance EraPParams era => DecCBOR (ConwayGovPredFailure era) where
     7 -> SumD ExpirationEpochTooSmall <! From
     8 -> SumD InvalidPrevGovActionIdsInProposals <! From
```

Comment:
I guess this constructor can be removed and replaced be the newer one, at the same position, instead of adding a new index to the serialization?

Reply:
Yes. Just saw your comment. I left the same comment :slightly_smiling_face: